### PR TITLE
add to profile after working on rust libs

### PR DESCRIPTION
**Pull request by zeeshanlakhani**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1,6 +1,76 @@
...
+
+### Related Skip-Ratchet Implementations
+
+- [Go][skip-go]
+- [Rust][skip-rust]
+- [Typescript][skip-ts]
+
+### Related Papers
+
+- [Skip Ratchet: A Hierarchical Hash System][paper] by Brooklyn Zelenka
```
Comment (expede) [inline profile/README.md:63]: Strange Loop: https://www.youtube.com/watch?v=3UjQd-JnMrQ
Paper: https://eprint.iacr.org/2022/1078

Comment (zeeshanlakhani) [reply profile/README.md:63]: ah, SL vid is out, good good!

Comment (zeeshanlakhani) [reply profile/README.md:63]: paper is linked already. I was looking at the talk name based on SL sessions! whoops

**Review by expede — APPROVED** [review]
Looks great to me! Thanks @zeeshanlakhani

**Pull request merged**